src/server/Document/component.jsx

```diff
@@ -73,6 +73,15 @@ const Document = ({
           />
         )}
         {scriptsAllowed && scriptTags}
+        {scriptsAllowed && (
+          <script
+            type="text/javascript"
+            /* eslint-disable-next-line react/no-danger */
+            dangerouslySetInnerHTML={{
+              __html: `document.documentElement.className = document.documentElement.className.replace(/\\bno-js\\b/,'js');`,
```

Comment:
> Just wondering how you were able to circumvent this?
@jamesdonoh adding this bit ensured that the `no-js` class when initially rendered would be replaced when js is enabled and when disabled, the code here won't run since it's in a `script`. More context on this is detailed in [the post](https://dylanvann.com/css-in-js/) shared by @tochwill